package.json

```diff
@@ -80,6 +80,7 @@
     "concurrently": "^3.5.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-react-intl": "^2.0.0",
```

Comment:
See earlier comment about `react-intl`. And for future notice, please also commit `yarn.lock` after changing something in `package.json` or already if some minor version got updated.